gradio/blocks.py

```diff
@@ -721,8 +721,11 @@ def __init__(
             else analytics.analytics_enabled()
         )
         if self.analytics_enabled:
-            t = threading.Thread(target=analytics.version_check)
-            t.start()
+            if wasm_utils.IS_WASM:
+                analytics.version_check()
```

Comment:
Should this also be called via asyncio? Otherwise could be slow down the launch of the app. Tbh this is probably the least important of the "analytics functions" so okay to skip it entirely if running is wasm mode

Reply:
Yes, thanks.
and `version_check()` also uses `requests.get` so I should have replaced it with `pyfetch` for the Wasm ver. as well.

Reply:
I looked its internals and found that `analytics.version_check()` compares the self package version and the latest version info obtained from the Web API (`https://api.gradio.app/pkg-version`), and warns users to upgrade the package.
However, it's not suitable for Gradio-lite because the Python package version is pinned for each Gradio-lite's version and users don't have control to upgrade the Python package version.
Doing the same thing for Gradio-lite's version makes sense, but we don't have the WebAPI for it for now.

So I simply deleted the function call of it here.

Reply:
Ah interesting, makes sense